packages/rich-text/src/component/event-listeners/copy-handler.js

```diff
@@ -30,10 +30,12 @@ export default ( props ) => ( element ) => {
 		}
 	}
 
-	element.addEventListener( 'copy', onCopy );
-	element.addEventListener( 'cut', onCopy );
+	const { defaultView } = element.ownerDocument;
+
+	defaultView.addEventListener( 'copy', onCopy );
+	defaultView.addEventListener( 'cut', onCopy );
```

Comment:
I believe this change produces a `TypeError: Cannot read properties of null (reading 'addEventListener')` error. I'm having trouble reproducing it manually, but it's consistent when running e2e tests.

```
npm run test:e2e -- -g "can be inserted after refresh"
```

## Screenshot
![CleanShot 2024-05-07 at 16 27 41](https://github.com/WordPress/gutenberg/assets/240569/ccb20da1-d648-499e-bc3d-838d9aa346e8)